### DISCUSSION TITLE
Addition of ACL to HAproxy

### DISCUSSION THREAD

**Discussion by rausub** [autobase-tech/autobase#852] [Ideas]
can we add option to have an ACL for haproxy in main.yml , default value can be none 
e.g.

    acl allowed_clients src 127.0.0.1 192.168.10.1 192.168.10.5 192.168.10.0/24
    tcp-request connection reject if !allowed_clients

**Comment by vitabaks** [comment]
Hi @rausub 

Feel free to suggest a PR if this is a really necessary option.
